packages/main/src/pages/protected/manage-workspace/ManageWorkspace.jsx

```diff
@@ -1,26 +1,26 @@
-import React, { useEffect, useState } from "react";
 import { AnimateSharedLayout, motion } from "framer-motion";
+import React, { useEffect, useState } from "react";
```

Comment:
Why move the react import to the second line when it's supposed to be at the very beginning in a js file

Reply:
I didnt import react or made use of useEffect or useState at all @JulianaSau is from somelse merge, or is prettier that organise it like so

Reply:
React should always be at the top level